### DISCUSSION TITLE
Suite / Unit Number not displaying

### DISCUSSION THREAD

**Discussion by ebart455987** [code4recovery/12-step-meeting-list#980] [Q&A] [answered]
How do I get the Suite/Unit number to display on the Meeting Guide App, meeting page?  
When I click on a particular meeting, the Suite number is missing.  However when I click on the Directions button the Unit number appears on the Map.  

![Screenshot 2022-11-30 at 12 52 41 PM](https://user-images.githubusercontent.com/55007577/204872082-a08761d6-d72d-4b68-b495-f329809b6465.png)
![Screenshot 2022-11-30 at 12 53 12 PM](https://user-images.githubusercontent.com/55007577/204872119-f30540e5-841a-49ca-a493-b15a763547a9.png)

**Comment by joshreisner** [comment, marked as answer]
@ebart455987 looks like you've got it displaying properly on your website: https://www.fortworthaa.org/meetings/wichita-falls/

please reach out to the General Service Office at appsupport@aa.org for questions about the Meeting Guide app.
